style.css

```diff
@@ -0,0 +1,36 @@
+
+  p{font-size: 2.5ch;}
+  h2{font-size: 25px;}
+  ul{font-size: 15px;}
+  button{background-color:burlywood}
+  button{font-size: 15px;}
```

Comment:
It's typically a good idea to use `em` over `px` because pixel size is different on every device. (i.e. one device might have a larger screen size than another and therefore have more pixels). Em is a way of creating a scalable size.

<img width="1193" alt="image" src="https://github.com/NikoTheGreeko/VillageWebsiteDrosia/assets/33900517/7f479470-af0d-438c-8308-ffb2cbc8a4f0">